lib/assets/codegens.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  'nodejs-requests': require('../../postman-code-generators/codegens/nodejs-request')
```

Comment:
Why is this in assets? 
And why have we kept postman-code-generators in the repository? 
Please remove this from the repo. 
Use the convert function exposed by the module: `convert('node', 'request', options, callback)`